src/getModule/parseInputs/parse.ts

```diff
@@ -16,7 +23,28 @@ export default function parse(
     const { tags } = input
     if (tags?.includes('any')) return any(arg)
 
-    if (tags?.includes('decimals')) return decimals(arg, extras)
```

Comment:
should be in this format decimals() function takes care of the logic so why not just include your logic inside decimals()?